Resources/Locale/en-US/reagents/meta/medicine.ftl

```diff
@@ -125,7 +125,7 @@ reagent-name-insuzine = insuzine
 reagent-desc-insuzine = Rapidly repairs dead tissue caused by electrocution, but cools you slightly. Completely freezes the patient when overdosed.
 
 reagent-name-necrosol = necrosol
-reagent-desc-necrosol = A necrotic substance that seems to be able to heal frozen corpses
+reagent-desc-necrosol = A necrotic substance that seems to be able to heal frozen corpses.
```

Comment:
is there a reason this punctuation was removed?

Reply:
You misread it, the punctuation was added by this, not removed